packages/tools/kadena-cli/src/utils/globalOptions.ts

```diff
@@ -195,6 +196,64 @@ export const globalOptions = {
       'Version of the kadena/devnet Docker image to use (e.g. "latest")',
     ),
   }),
+  //Simulate
+  simulateNoAccounts: createOption({
+    key: 'accounts' as const,
+    prompt: simulatePrompts.simulateNoAccountsPrompt,
+    validation: z.number(),
+    option: new Option(
+      '-a, --accounts <accounts>',
+      'Enter the amount of accounts to be created in the simulation.',
+    ),
+  }),
+  simulateTransferInterval: createOption({
+    key: 'interval' as const,
+    prompt: simulatePrompts.simulateTransferIntervalPrompt,
+    validation: z.number(),
+    option: new Option(
+      '-i, --interval <interval>',
+      'Enter the transfer interval in milliseconds.',
+    ).argParser((value) => parseInt(value, 10)),
+  }),
+
+  simulateMaxAmount: createOption({
+    key: 'maxAmount' as const,
+    prompt: simulatePrompts.simulateMaxAmountPrompt,
+    validation: z.number(),
+    option: new Option(
+      '-x, --max-amount <maxAmount>',
+      'Enter the max transfer amount per single transaction (coin).',
+    ),
+  }),
+
+  simulateTokenPool: createOption({
+    key: 'tokenPool' as const,
+    prompt: simulatePrompts.simulateTokenPoolPrompt,
+    validation: z.number(),
+    option: new Option(
+      '-p, --token-pool <tokenPool>',
+      'Enter the total token pool (coin).',
+    ),
+  }),
+  simulateSeed: createOption({
+    key: 'seed' as const,
+    prompt: simulatePrompts.simulateSeedPrompt,
+    validation: z.string(),
+    option: new Option(
+      '-s, --seed <seed>',
+      'Enter the seed for the simulation.',
+    ),
+  }),
+  simulateLogFolder: createOption({
+    key: 'logFolder' as const,
+    prompt: simulatePrompts.simulateLogFolderPrompt,
+    validation: z.string(),
+    option: new Option(
+      '-l, --log-folder <logFolder>',
+      'Specify the directory where the log file will be generated.',
+    ),
+  }),
```

Comment:
Having also this in mind: https://github.com/kadena-community/kadena.js/pull/1476#discussion_r1453533628

What should be the go-to behavior? The first option should be a one letter flag when possible (since the second already states what it is) or should we keep one letter flags for booleans and common options?